.github/workflows/release-fastlane-android.yml

```
@@ -38,9 +38,9 @@ jobs:
         uses: google-github-actions/get-secretmanager-secrets@v2
         with:
           secrets: |-
-            ANDROID_RELEASE_KEYSTORE:projects/1027349420744/secrets/ANDROID_RELEASE_KEYSTORE
-            CELO_RELEASE_KEY_PASSWORD:projects/1027349420744/secrets/CELO_RELEASE_KEY_PASSWORD
-            CELO_RELEASE_STORE_PASSWORD:projects/1027349420744/secrets/CELO_RELEASE_STORE_PASSWORD
+            ANDROID_RELEASE_KEYSTORE:projects/1027349420744/secrets/ANDROID_VALORA_MOBILESTACK_RELEASE_KEYSTORE
+            MOBILESTACK_RELEASE_KEY_PASSWORD:projects/1027349420744/secrets/VALORA_MOBILESTACK_RELEASE_KEY_PASSWORD
+            MOBILESTACK_RELEASE_STORE_PASSWORD:projects/1027349420744/secrets/VALORA_MOBILESTACK_RELEASE_STORE_PASSWORD
```

Shorter names, I don't feel we need Mobile Stack in those names.
```suggestion
            ANDROID_RELEASE_KEYSTORE:projects/1027349420744/secrets/VALORA_RELEASE_KEYSTORE
            RELEASE_KEY_PASSWORD:projects/1027349420744/secrets/VALORA_RELEASE_KEY_PASSWORD
            RELEASE_STORE_PASSWORD:projects/1027349420744/secrets/VALORA_RELEASE_STORE_PASSWORD
```

We could even use the same password for the store and the key. To simplify.